src/Presentation/Console/Command/InitProjectCommand.php

```diff
@@ -29,9 +29,9 @@ class InitProjectCommand extends Command
     protected const NAME = 'sdk:init:project';
 
     /**
-     * @var \SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver
      */
-    protected CliValueReceiver $cliValueReceiver;
+    protected CliReceiver $cliValueReceiver;
```

Comment:
It must be depended on the abstraction. Please change it to the `ReceiverInterface` interface.